src/index.html

```diff
@@ -1,12 +1,412 @@
+
+            <div class="card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__review-txt">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+
+              <p class="card__price-amount">$2,199</p>
+            </div>
+
+            <div class="card__button">
+              <a href="#" class="card__button-link">
+                Buy
+              </a>
+            </div>
+          </li>
+
+          <li class="card__item">
+            <div class="card__image">
+              <img
+              class="card__image-img"
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27inc Retina">
+            </div>
+
+            <h2 class="card__product-name">
+              APPLE A1419 iMac 27" Retina
+              <br>
+              5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="card__product-id">Product code: 195434</p>
+
+            <div class="card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__review-txt">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+
+              <p class="card__price-amount">$2,199</p>
+            </div>
+
+            <div class="card__button">
+              <a href="#" class="card__button-link">
+                Buy
+              </a>
+            </div>
+          </li>
+
+          <li class="card__item">
+            <div class="card__image">
+              <img
+              class="card__image-img"
+              src="images/imac.jpeg"
+              alt="APPLE A1419 iMac 27inc Retina">
+            </div>
+
+            <h2 class="card__product-name">
+              APPLE A1419 iMac 27" Retina
+              <br>
+              5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="card__product-id">Product code: 195434</p>
+
+            <div class="card__review">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__review-txt">Reviews: 5</p>
```

Comment:
if you started using `-txt` naming convention for classes you should follow this rule or change it to `-text` as you have it in `card__price-text`